packages/apidom-ls/src/config/openapi/openapi3_1/lint/webhooks--type.ts

```diff
@@ -0,0 +1,18 @@
+// OPENAPI3_1_WEBHOOKS_TYPE
+import ApilintCodes from '../../../codes';
+import { LinterMeta } from '../../../../apidom-language-types';
+
+const webhooksLint: LinterMeta = {
+  code: ApilintCodes.OPENAPI3_1_OPEN_API_FIELD_WEBHOOKS_TYPE,
```

Comment:
Note also that if changed to `webhooks--values-type.ts` you need to amend to name of the error code as well.

```
OPENAPI3_1_OPEN_API_FIELD_WEBHOOKS_TYPE -> OPENAPI3_1_OPEN_API_FIELD_WEBHOOKS_VALUES_TYPE
```